docs/user/aws/limits.md

```diff
@@ -5,29 +5,88 @@ You can find a comprehensive list of the default AWS service limits published he
 [AWS Service Limits][service-limits]
 
 Below, we'll identify OpenShift cluster needs and how those impact some of those limits.
+We will also use [aws cli](https://aws.amazon.com/cli/) and [jq](https://stedolan.github.io/jq/) to try and explan
+where limits are set (viewable) and how to show your current usage of resources.
+
+Many commands below are based on [Trusted Advisor](https://aws.amazon.com/premiumsupport/technology/trusted-advisor/)
+and often this can be used to print both the limit and the usage for a service. However to check those limits/usage
+you need to first identify a check_id. These check_id's can be pulled from:
+
+```sh
+$ aws support describe-trusted-advisor-checks --language en | jq -c '.checks[] | {ID: (.id), Name: .name, Category: .category}'
+```
+
+Should you need to increase a limit (with any of the following) you need to open a ticket with AWS support, according to
+[AWS Service Limits](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html).
+
+### Example: Using N. Virginia (us-east-1) to increase a limit for a VPC Elastic IP
+
+To use N. Virginia (us-east-1) for a new cluster, please submit a limit increase for VPC Elastic IPs similar to the
+following in the support dashboard (to create more than one cluster, a higher limit will be necessary):
+
+![Increase Elastic IP limit in AWS](images/support_increase_elastic_ip.png)
 
 ## S3
 
 There is a default limit of 100 S3 buckets per account. The installation creates a bucket temporarily. Also, the
 registry component creates a permanent bucket. This will limit the number of clusters per account to 99 initially. To
 support additional clusters, you must open a support case with AWS.
 
+You can not view the limits for S3 buckets, however you can view how many buckets you are using (what the name and
+creation date is), by running the following:
+
+```sh
+$ aws --region us-east-1 s3api list-buckets | jq -c '.Buckets[]'
```

Comment:
Non-`jq` form (one less dependency) with the creation date and name on one line for easy counting:

```console
$ aws --region us-east-1 s3api --query 'join(`\n`, @.Buckets[].join('"' '"', [@.CreationDate, @.Name]))' --output text list-buckets
2019-01-18T06:25:50.000Z cloud-trail-test-clayton
2019-01-18T06:30:13.000Z aws-athena-query-results-460538899914-us-east-1
2019-02-18T11:47:32.000Z image-registry-us-east-1-9e591a5de9c24ca0a5006aa16fef32c7-f52a
2019-02-19T09:10:50.000Z image-registry-us-east-1-65cb42acad1b4ef59d18edeefbb9418b-3ca2
...
```

That's a bit awkward, but it's easier if we can drop the creation dates:

```console
$ aws --region us-east-1 s3api --query 'join(`\n`, @.Buckets[].Name)' --output text list-buckets
aws-athena-query-results-460538899914-us-east-1
cloud-trail-test-clayton
image-registry-us-east-1-79e02d69c95e4789b63e521d2d2d6322-5964
...
```

Also, both of these are showing buckets in a single region, when it sounds like the limit is [per-account][1], not per-region.

[1]: https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html

Reply:
I fixed the command to be (account-based) and to remove the jq dependency.

Reply:
@wking take a look at what I did here (I feel like I am missing something).